src/components/ConfigCosts/index.tsx

```diff
@@ -0,0 +1,108 @@
+import { useEffect, useMemo, useState } from "react";
+import { FaDollarSign } from "react-icons/fa";
+import CollapsiblePanel from "../CollapsiblePanel";
+import { Loading } from "../Loading";
+
+export function FormatCurrency({ value }: { value: number | string }) {
+  const amount = useMemo(() => {
+    const parsedValue = typeof value === "string" ? parseFloat(value) : value;
+    if (value > 1000) {
+      return (parsedValue / 1000).toFixed(1) + "k";
+    }
+    if (parsedValue > 100) {
+      return parsedValue.toFixed(0);
+    }
+    if (parsedValue > 10) {
+      return parsedValue.toFixed(1);
+    }
+    if (parsedValue > 0.01) {
```

Comment:
I think lets always round up to 2 decimal places

Reply:
@moshloop done